### PR TITLE
Create protobuf for wire contracts

### PR DESCRIPTION
**Pull request by NonStatic2014**
Create protobuf for wire contracts and update build

### CODE REVIEW

File: cmake/onnxruntime_hosting.cmake [hunk 1]
```diff
@@ -37,6 +61,10 @@ target_link_libraries(${PROJECT_NAME} PRIVATE
         onnxruntime_graph
         onnxruntime_common
         onnxruntime_mlas
+        onnx
```
Comment (tmccrmck) [inline cmake/onnxruntime_hosting.cmake]: These are all in onnxruntime_EXTERNAL_LIBRARIES

File: onnxruntime/hosting/protobuf/model_metadata.proto [hunk 2]
```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
```
Comment (tmccrmck) [inline onnxruntime/hosting/protobuf/model_metadata.proto:1]: I'm confused... are these from the onnx repo proto3 definitions? Did you just split them into multiple files?

Comment (tmccrmck) [reply onnxruntime/hosting/protobuf/model_metadata.proto:1]: Resolved

**Review by tmccrmck — APPROVED** [review]

File: cmake/onnxruntime_hosting.cmake (hunk 1 above)
Comment (NonStatic2014) [reply cmake/onnxruntime_hosting.cmake]: Removed

**Pull request merged**